### PR TITLE
 JAMES-3319 Actual Message & Attachment deletion for mailbox/cassandra 

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
```
CassandraMessageMapperTest$StatementLimitationTests.findInMailboxLimitShouldLimitProjectionReadCassandraQueries:200 
[dfb80d1621e2b30c19e543ef96008ce8fc431ce6] Expected size:<2> but was:<0> in:
```

test this please

**Comment by chibenwa** [comment]
It looks like I broke something....

File: mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerTest.java [hunk 1]
```diff
@@ -159,6 +161,28 @@ void deleteMailboxShouldUnreferenceMessageMetadata(CassandraCluster cassandraClu
...
         }
 
+        @Test
+        void deleteMailboxShouldDeleteMessageAndAttachmentBlobs(CassandraCluster cassandraCluster) throws Exception {
+            inboxManager.appendMessage(MessageManager.AppendCommand.builder()
+                .build(ClassLoaderUtils.getSystemResourceAsByteArray("eml/emailWithOnlyAttachment.eml")), session);
+
+            mailboxManager.deleteMailbox(inbox, session);
+
+            assertThat(cassandraCluster.getConf().execute(select().from(BlobTables.DefaultBucketBlobTable.TABLE_NAME)))
```
Comment (remk) [inline mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerTest.java:171]: We should check the deletion of the messages too ?

File: mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerTest.java [hunk 2]
```diff
@@ -159,6 +161,28 @@ void deleteMailboxShouldUnreferenceMessageMetadata(CassandraCluster cassandraClu
...
+        }
+
+        @Test
+        void deleteMessageShouldDeleteMessageAndAttachmentBlobs(CassandraCluster cassandraCluster) throws Exception {
+            AppendResult appendResult = inboxManager.appendMessage(MessageManager.AppendCommand.builder()
+                .build(ClassLoaderUtils.getSystemResourceAsByteArray("eml/emailWithOnlyAttachment.eml")), session);
+
+            inboxManager.delete(ImmutableList.of(appendResult.getId().getUid()), session);
+
+            assertThat(cassandraCluster.getConf().execute(select().from(BlobTables.DefaultBucketBlobTable.TABLE_NAME)))
```
Comment (remk) [inline mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerTest.java:182]: We should check the deletion of the message too ?

File: mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerTest.java (hunk 1 above)
Comment (chibenwa) [reply mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerTest.java:171]: That is done in previous already existing tests.

File: mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerTest.java (hunk 2 above)
Comment (chibenwa) [reply mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerTest.java:182]: That is done in previous already existing tests.

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**